StructureDefinition/England-Task.xml

```diff
@@ -0,0 +1,304 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="England-Task" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/England-Task" />
+  <version value="1.0.0" />
+  <name value="EnglandTask" />
+  <title value="England Task" />
+  <status value="draft" />
+  <date value="2023-06-01" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="This profile defines the NHS England constraints and extensions on the FHIR UK Core resource UK Core Task." />
```

Comment:
Description isn't using the agreed template. Needs to be the template that includes the link to the Simplifier page.

Reply:
Changed the description